Logo/README.md

```diff
@@ -0,0 +1,29 @@
+# ClassicPress logos
+
+The main files are SVG ones.
+
+# Automatic conversion
+
+In the autogenerate folder you'll find automatically converted assets.
+
+The script checks that:
+
+- the paths are closed;
+- SVG title is file name (without extension).
+
+## Conversion
+In the Logo directory source the script:
+```
+$ . bin/generate-assets.sh
+```
```

Comment:
The script needs to be written so that it works from any directory, and it should be ran rather than sourced (for people who are using zsh for example).

Reply:
Ok, done!